tests/by-util/test_ls.rs

```diff
@@ -2099,6 +2099,32 @@ fn test_ls_order_time() {
     }
 }
 
+#[test]
+fn test_ls_order_mtime() {
+    use std::time::SystemTime;
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    let f = at.make_file("test-3");
+    f.set_modified(SystemTime::now()).unwrap();
+    let f = at.make_file("test-4");
+    f.set_modified(SystemTime::now()).unwrap();
```

Comment:
I would give those vars a different name than `f`. It should allow you to get rid of the `make_file` calls on lines 2119 and 2122.

Reply:
well done, thanks again :)